ts/Core/Chart/GanttChart.ts

```diff
@@ -147,27 +146,29 @@ class GanttChart extends Chart {
                 defaultLinkedTo = 0;
             }
             return merge(
-                defaultOptions.xAxis,
-                { // defaults
+                // Defaults
+                {
                     grid: {
                         enabled: true
                     },
                     opposite: true,
                     linkedTo: defaultLinkedTo
                 },
-                xAxisOptions, // user options
-                { // forced options
+                // User options
+                xAxisOptions,
+                // Forced options
+                {
                     type: 'datetime'
                 }
             );
         });
 
-        // apply Y axis options to both single and multi y axes
+        // Apply Y axis options to both single and multi y axes
         options.yAxis = (splat(userOptions.yAxis || {})).map((
             yAxisOptions: YAxisOptions
         ): YAxisOptions => merge(
-            defaultOptions.yAxis, // #3802
-            { // defaults
+            // Defaults
```

Comment:
Without `defaultOptions.yAxis` this now produces a regression on #3802:
Example https://jsfiddle.net/6rj2s4xh/ where second and third chart should have axis to the left, in both chart and navigator.

Reply:
Ok. This probably requires that we change the (actual) default value of `opposite` to false, so that we can check against `defaultOptions` here: 

https://github.com/highcharts/highcharts/blob/b74c01fe0b1df5fe6bb4e424ffbb415db9a614aa/ts/Core/Chart/StockChart.ts#L135